src/main/java/teammates/logic/core/GoogleCloudLoggingService.java

```diff
@@ -108,6 +120,45 @@
         return errorLogs;
     }
 
+    @Override
+    public QueryLogsResults queryLogs(List<String> severities, Instant startTime, Instant endTime,
+            Integer pageSize, String pageToken) throws LogServiceException {
+        LogSearchParams logSearchParams = new LogSearchParams()
+                .addLogName(STDOUT_LOG_NAME)
+                .addLogName(STDERR_LOG_NAME)
+                .setSeverities(severities)
+                .setStartTime(startTime)
+                .setEndTime(endTime);
+
+        PageParams pageParams = new PageParams(pageSize, pageToken);
+
+        Page<LogEntry> logEntriesInPage = getLogEntries(logSearchParams, pageParams);
+        List<GeneralLogEntry> logEntries = new ArrayList<>();
+        for (LogEntry entry : logEntriesInPage.getValues()) {
+            String logName = entry.getLogName();
+            Severity severity = entry.getSeverity();
+            String trace = entry.getTrace();
+            com.google.cloud.logging.SourceLocation sourceLocation = entry.getSourceLocation();
```

Comment:
> There is a self-defined class having the same name as `SourceLocation` of google cloud logging and `GoogleCloudLoggingService.java` uses self-defined class as well.

The said class is not imported here and is called as part of `GeneralLogEntry`, which means this fully-qualified name is still not needed.

Reply:
There is another class `com.google.appengine.logging.v1.SourceLocation;` that is used here